xtask/src/commands/test.rs

```diff
@@ -34,15 +34,7 @@ impl Test {
         );
 
         // NOTE: it worked nicely on GitHub Actions but it hangs on CircleCI on Windows
```

Comment:
This comment can now be removed \o/

Reply:
Fixed in: [62c4943](https://github.com/apollographql/router/pull/850/commits/62c494364052461883d0e9f36a103d62545dfad6)